TESTS/mbedmicro-rtos-mbed/heap_and_stack/main.cpp

```diff
@@ -39,9 +39,9 @@ static const int test_timeout = 30;
 // Malloc fill pattern
 #define MALLOC_FILL                 0x55
 
-extern uint32_t mbed_heap_start;
+extern unsigned char *mbed_heap_start;
```

Comment:
?
```suggestion
extern uint8_t *mbed_heap_start;
```

Reply:
I corrected the type to what it is set to in `mbed_boot.h`,  https://github.com/ARMmbed/mbed-os/blob/master/rtos/source/TARGET_CORTEX/mbed_boot.h/#L57

Reply:
Does it make it right?